frontend/packages/pipelines-plugin/src/const.ts

```diff
@@ -4,3 +4,8 @@ export const CLUSTER_PIPELINE_NS = 'openshift';
 export const PIPELINE_RUNTIME_LABEL = 'pipeline.openshift.io/runtime';
 export const PIPELINE_RUNTIME_VERSION_LABEL = 'pipeline.openshift.io/runtime-version';
 export const PIPELINE_STRATEGY_LABEL = 'pipeline.openshift.io/strategy';
+export const PAC_SECRET_NAME = 'pipelines-as-code-secret';
+export const PAC_EL_ROUTE_NAME = 'el-pipelines-as-code-interceptor';
+export const PIPELINES_NS = 'openshift-pipelines';
```

Comment:
nit: There is already a const in pointing to Openshift Pipelines Namespace [
PIPELINE_NAMESPACE](https://github.com/openshift/console/blob/master/frontend/packages/pipelines-plugin/src/components/pipelines/const.ts#L68-L69), we can instead